dxr/plugins/python/indexers.py

```diff
@@ -25,14 +26,27 @@
 }
 
 
+class _FileToIgnore(object):
+    """A file that we don't want to bother indexing, usually due to
+    syntax errors.
+
+    """
+    def is_interesting(self):
+        return False
+FILE_TO_IGNORE = _FileToIgnore()
+
+
 class TreeToIndex(TreeToIndexBase):
     def __init__(self, *args, **kwargs):
         super(TreeToIndex, self).__init__(*args, **kwargs)
 
-        self.python_path = getattr(self.tree, 'plugin_python_path', self.tree.source_folder)
+        self.python_path = self.plugin_config.python_path
```

Comment:
Removing that default was intentional; I think it's a good thing to force projects to be explicit about where the modules start in their project. 

Happy to change that if you disagree, I just wanted to make sure you knew it was intentional.